packages/material-ui-styles/src/index.js

```diff
@@ -0,0 +1,6 @@
+export { default as install } from './install';
+export { default as ThemeProvider } from './ThemeProvider';
+export { default as StylesProvider } from './StylesProvider';
+export { default as withStyles } from './withStyles';
+export { default as withTheme } from './withTheme';
+export { default as createStyles } from './createStyles';
```

Comment:
`createStylesHook` to avoid conflict? Or rename the old `createStyles` to `createTypedStyles`?  I prefer `createTypedStyles` more. Makes the intention clearer.

Reply:
I have changed the wording to avoid the conflict.